app/controllers/ramen_stores_controller.rb

```diff
@@ -27,6 +30,6 @@ def show
 
   private
     def ramen_store_params
-      params.require(:ramen_store).permit(:name, :sale, :address, :parking_space, :phone_number, :sns, :prefecture, :content)
+      params.require(:ramen_store).permit(:name, :sale, :address, :parking_space, :phone_number, :sns, :content, :prefecture_id, ramen_store_menu_attributes:[:id, :name, :price])
```

Comment:
これで保存されたりとかしますかね？
```suggestion
      params.require(:ramen_store).permit(:name, :sale, :address, :parking_space, :phone_number, :sns, :content, :prefecture_id, ramen_store_menus_attributes:[:id, :name, :price])
```

あと現状の状態でサーバーに送られて来ているパラメーターなんかもわかると嬉しいです！